### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by Panupong-Tongtawach**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "CI=true react-scripts test --env=jsdom",
```
Comment (nutstick) [inline package.json]: Not add CI here, add at travis config

File: package.json [hunk 2]
```diff
@@ -53,4 +53,4 @@
       "git add"
     ]
   }
-}
+}
```
Comment (nutstick) [inline package.json:56]: Why no new line?

**Review by nutstick — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Panupong-Tongtawach) [reply package.json:56]: look like lint is not apply to `.json`

Comment (nutstick) [reply package.json:56]: Should need as only file that is json is `package.json`. Self review should be enough.

**Pull request merged**